bff/config.json

```diff
@@ -66,6 +66,16 @@
         "io_write_throughput": "sum(cc_system_disk_write_bytes_per_second{sys_id=~\"$sys_id\",sys_scope=~\"$sys_scope\",sys_type=~\"$sys_type\"})",
         "free_storage_space": "sum(cc_system_diskpartition_free_bytes{sys_id=~\"$sys_id\",sys_scope=~\"$sys_scope\",sys_type=~\"$sys_type\"})"
       }
+    },
+    "/v1/snapshots": {
+      "params": [
+        "limit",
+        "type"
+      ],
+      "options": {
+        "limit": "$limit",
+        "type": "$type"
+      }
```

Comment:
You don't need this change. This file contains the APIs that return data from prometheus.

```suggestion
```

Reply:
This doesn't return data from Prometheus but it does return. Where would you suggest it goes?

Reply:
`bff/config.json` is our automated API creation tool for the data returned from prometheus. For everything else, like your `/v1/snapshots`, you'd explicitly create an endpoint in `server.go`, as you've already done. This change is unnecessary and doesn't do anything (and is dead code).

Reply:
I see. Ok...